cpmonitor/models.py

```diff
@@ -358,13 +358,18 @@ class Meta:
         help_text="Nur falls im Klimaaktionsplan angegeben.",
     )
 
-    responsible_organ = models.TextField(
+    responsible_organ = models.CharField(
         "Verantwortliches Organ",
         blank=True,
+        max_length=100,
+    )
+
+    responsible_organ_explanation = models.TextField(
+        "Erklärungstext zum verantwortlichen Organ",
+        blank=True,
         help_text="""
-            <p>Genauer Name des verantwortlichen Gremiums oder der verantwortlichen Behörde.</p>
-            <p>Möglichst mit Ansprechperson, Kontaktdaten und was sonst notwendig ist, um Informationen einzuholen.</p>
-            <p>Diese Informationen könnten später in separate Felder aufgeteilt werden.</p>
+            <p>Eventuell Ansprechperson oder Kontaktdaten, wenn diese öffentlich zugänglich sind.</p>
+            <p>Alternativ eine Begründung, warum es an diesem Organ hängt.</p>
```

Comment:
Why did you remove some of the information from the help_text? I think it is still very much valid, that we intend to add contact information and that we intend to put more structure into the data to enable something like frag-den-staat.de. (See my prototype fund application.)

I like that you added the possibility of a "Begründung". But I would rather add it instead of an alternative:

I would also leave "Genauer Name" in there. It is quite likely (at least here in Bremen. ;-)) that it is longer than anything that fits in the 100 characters.

```suggestion
            <p>Genauer Name des verantwortlichen Gremiums oder der verantwortlichen Behörde.</p>
            <p>Möglichst mit Ansprechperson, Kontaktdaten und was sonst notwendig ist, um Informationen einzuholen.</p>
            <p>Diese Informationen könnten später in separate Felder aufgeteilt werden.</p>
            <p>Ggf. auch eine Begründung, warum es an diesem Organ hängt.</p>
```

Reply:
- exact name: removed because there is an extra field for this -> should I make the field bigger then? (I would not duplicate the information)
- contact info: changed because I was afraid we might get into legal trouble if we put private addresses here
- extra fields: removed because what we might or might not do should not bother the pilots (I did not even get the Frag den Staat hint) and in any case I would not put hypothetical things into the code

Reply:
Lets discuss this in person.

Some information about "exact name" as background:
In Bremen we had something called "SKUMS". I would like to display only "SKUMS" on the Task page. The full / exact name is

"Senatorin für Klimaschutz, Umwelt, Mobilität, Stadtentwicklung und Wohnungsbau"

which might or might not be below 100 characters. (I am too lazy to count right now...) But I find it too long to be shown on the Task page and also beside the point, since everyone politically interested in Bremen knows, what "SKUMS" means.